### PR TITLE
Like functionality

### PR DESCRIPTION
**Pull request by sameep-s**
Like Functionality.

- click like button to add a video to liked videos.
- Click again to remove from liked videos.
- view all liked videos
- remove directly from liked videos.
- Go to single product page from the liked videos.

### CODE REVIEW

File: src/components/VideoComponent/VideoComponent.jsx [hunk 1]
```diff
@@ -3,23 +3,28 @@ import './videoComponent.css';
...
 
+    function addToLikeHandler() {
+        dispatch_data({ type: "LIKE_VIDEO", payload: { video: video } })
+    }
+
+    const isLiked = isVideoPresentInLiked(state_data.liked, _id);
+
+
+
+
```
Comment (munjalSaksham71) [inline src/components/VideoComponent/VideoComponent.jsx]: Remove this extra spacing

**Review by munjalSaksham71 — APPROVED** [review]
Functionality works as expected. Just a suggestion: use prettier to format document. And remove extra lines from code.

File: src/components/VideoComponent/VideoComponent.jsx (hunk 1 above)
Comment (sameep-s) [reply src/components/VideoComponent/VideoComponent.jsx]: Removed

**Review by parthgabani777 — APPROVED** [review]
looks good!

**Pull request merged**